src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+      "
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/main.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body page__body">
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac image"
+        class="card__image"
+      >
+
+      <ul class="card__body">
+        <li class="card__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </li>
+
+        <li class="card__code">
+          Код товара: 195434
+        </li>
+
+        <li class="card__stars">
+          <ul class="rate">
+            <li class="rate__stars rate__stars--4">
+              <div class="rate__image"></div>
+              <div class="rate__image"></div>
+              <div class="rate__image"></div>
+              <div class="rate__image"></div>
+              <div class="rate__image"></div>
+            </li>
+          </ul>
+          <div class="card__vote-votes">Отзывов: 5</div>
+        </li>
+
+        <li class="card__price">
+          <div class="card__price-text">Цена:</div>
+          <div class="card__price-number">69 999 грн</div>
+        </li>
+
+        <li class="card__purchase" data-qa="hover">
+          <a class="card__purchase-link"  href="#">Купить</a>
+        </li>
+      </ul>
```

Comment:
why do you use list and list elements here? that's not the place to use list. remove them